Modules/AirCastingStyling/Sources/AirCastingStyling/FollowButtonStyle.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  File.swift
+//  
+//
+//  Created by Lunar on 19/07/2021.
+//
```

Comment:
those headers don't seem consistent with the rest of the codebase

Reply:
Same, they're the same as other headers in the AirCastingStyling module

Reply:
Can't see really a difference too, could you please point it out?

Reply:
Ok, didn't consider `AirCastingStyling`. We should change it some day to the same format we're using in the app module (no File.swift nonsense)